array-destructuring/exercise-2/exercise.js

```diff
@@ -70,3 +70,15 @@ let hogwarts = [
     occupation: "Teacher",
   },
 ];
+
+hogwarts.forEach(({ firstName, lastName, house }) => {
+  if (house === "Gryffindor") {
+    console.log(`${firstName} ${lastName}`);
+  }
+});
+
+hogwarts.forEach(({ firstName, lastName, pet, occupation }) => {
+  if (occupation === "Teacher" && pet) {
+    console.log(`${firstName} ${lastName}`);
+  }
+});
```

Comment:
As above, this is fine, but an alternative would be to split out the filtering from the printing:

```js
hogwarts
  .filter(({ occupation, pet }) => occupation === "Teacher" && pet)
  .forEach(({ firstName, lastName }) => {
    console.log(`${firstName} ${lastName}`);
  });
```